chains/evm/executor/executor.go

```diff
@@ -27,6 +27,12 @@ import (
 )
 
 const TRANSFER_GAS_COST = 200000
+const MAX_TRANSACTION_COST = 25000000
```

Comment:
25 million? That doesn't sound like the correct number.

Reply:
Current max block size on Ethereum is 30 million.
I reduced to be a bit less.
Though now that you mention it maybe makes sense to make configurable if some EVM has a different block size (for sure it has)

Reply:
It actually makes sense now, but you are right having this as a configurable property per chain probably makes the most sense (25mil can be the default value).

Reply:
I used the `gasLimit` field already in the configuration because we don't use that anymore and it made sense